src/pages/Payment/PaymentOrderDetails.js

```diff
@@ -16,32 +44,75 @@ class PaymentOrderDetails extends React.Component {
             <h3>국내배송상품 주문내역</h3>
           </header>
           <table className="orderDetailsTable">
-            {ORDER_DETAILS_LIST.map(list => {
-              const { key, content } = list;
-              return (
-                <th className="tableStyle" key={key}>
-                  {content}
-                </th>
-              );
-            })}
-            <tr>
-              <td className="tableline">{'img'}</td>
-              <td className="tableline">{'product'}</td>
-              <th className="tableline">{'price'}</th>
-              <th className="tableline">{'quantity'}</th>
-              <th className="tableline">{'delivery'}</th>
-              <th className="tableline">{'delivery fee'}</th>
-              <th className="tableline">{'total'}</th>
-            </tr>
-            <th colspan="9" className="totalPrice">
-              <strong>기본배송</strong> 상품구매금액 {''}+ 배송비 {''} -
-              상품할인금액 {''} = 합계:
-              <span>
-                {''}
-                <strong>원</strong>
-              </span>
-            </th>
+            <thead>
+              <tr>
+                {ORDER_DETAILS_LIST.map(list => {
+                  const { key, content } = list;
+                  return (
+                    <th className="tableStyle" key={key}>
+                      {content}
+                    </th>
+                  );
+                })}
+              </tr>
+            </thead>
+            <tbody>
+              {this.state.paymentList.map(el => {
+                return (
+                  <tr key={el.order_number}>
+                    <td className="tableline">
+                      {el.information[0].product_image}
```

Comment:
```suggestion
              {this.state.paymentList.map(payment => {
                return (
                  <tr key={payment.order_number}>
                    <td className="tableline">
                      {star > el.information[0].product_image}
```
- el 같은 이름은 앞으로 내 인생에 없다고 생각하시고 앞으로는 꼭 구체적인 이름으로 변수명 지어주세요...
- 이건 이 PR 전반적인 궁금함인데 information의 0번 인덱스만 사용하는 이유가 특별히 있나요? 반드시 0번 인덱스만 사용되나요?

Reply:
변수명 수정했습니다..! information 안에 있는 배열의 객체에 접근해서 값을 가져오려고 했더니 몇번째 배열인지 지정해줘야 접근가능하여 [0] 번째를 따로 지정해주었습니다.